core/src/test/java/com/novoda/merlin/receiver/ConnectivityReceiverShould.java

```diff
@@ -57,10 +64,10 @@ public void notNotifyTheMerlinServiceOnNonConnectivityIntents() throws Exception
 
     @Test
     public void notifyTheMerlinServiceOnValidConnectivityIntents() throws Exception {
-        Intent intent = new Intent();
-        intent.setAction(ConnectivityManager.CONNECTIVITY_ACTION);
+        Intent mock = mock(Intent.class);
```

Comment:
I think this is a valid candidate for mocking - both the context and the intent are being called from outside of our system (outside of our class)